lxd/api_1.0.go

```diff
@@ -128,8 +128,11 @@ var api10 = []APIEndpoint{
 	warningCmd,
 	metricsCmd,
 	identitiesCmd,
-	identitiesByAuthenticationMethodCmd,
-	identityCmd,
```

Comment:
Sorry for this, I am having a hard time grasping why this separation is needed.
Could you please elaborate more on that?

Reply:
I'll add more detail to the commit message but this is so that we can have `POST /1.0/auth/identities/tls` for fine-grained TLS identity creation without `POST /1.0/auth/identities/oidc`